src/main/resources/external-policies-api.yaml

```diff
@@ -229,6 +252,94 @@ definitions:
+      result:
+        type: array
+        description: Array of matching items
+        items:
+          title: Policy Resource Schema
+          description: Json schema for the policy resource
+          type: object
+          properties:
+            name:
+              type: string
+              title: Name
+              description: String matching the name of the application
+            active:
+              type: boolean
+              title: Active flag
+              description: 'Boolean indicating whether OpenAM considers the policy active for evaluation purposes, defaults to false'
+            description:
+              type: string
+              title: Description
+              description: String describing the policy
+            applicationName:
+              type: string
+              title: Application name
+              description: 'String containing the application name, such as "iPlanetAMWebAgentService", or "mypolicyset"'
+            actionValues:
+              $ref: '#/definitions/ActionMap'
+            resources:
+              type: array
+              items:
+                type: string
+              title: Resources
+              description: List of the resource name pattern strings to which the policy applies. Must conform to the pattern templates provided by the associated Managing Resource Types resource type
+            subject:
+              $ref: '#/definitions/SubjectMap'
+            condition:
+              $ref: '#/definitions/ConditionMap'
+            resourceTypeUuid:
+              type: string
+              title: Resource Type UUID
+              description: The UUIDs of the resource type associated with the policy
+            resourceAttributes:
+              type: array
+              items:
+                type: object
+              title: Resource Attributes
+              description: List of attributes to return with decisions. These attributes are known as response attributes
+            lastModifiedBy:
+              type: string
+              title: Last Modified By
+              description: A string containing the universal identifier DN of the subject that most recently updated the policy
+            lastModifiedDate:
+              type: string
+              title: Last Modified date
+              description: 'An integer containing the last modified date and time, in number of seconds'
+            createdBy:
+              type: string
+              title: Created By
+              description: A string containing the universal identifier DN of the subject that created the policy
+            creationDate:
+              type: string
+              title: Creation Date
+              description: 'An integer containing the creation date and time, in number of seconds'
+      resultCount:
+        type: integer
+        format: int32
+        default: 0
+        description: Count of individual result items
+      pagedResultsCookie:
```

Comment:
you can leave out everything thats involved in pagination as we dont support it with this endpoint